README.rst

```diff
@@ -293,6 +293,46 @@ use the .rockspec.
 
 `Custom template example <https://github.com/tarantool/tt/blob/master/doc/examples.rst#working-with-application-templates>`_
 
+Working with tt daemon (experimental feature)
+---------------------------------------------
+
+``tt_daemon.yaml`` file format:
+
+.. code-block:: yaml
+
+  daemon:
+        run_dir: path
+        log_dir: path
+        log_maxsize: num (MB)
+        log_maxage: num (Days)
+        log_maxbackups: num
+        log_file: string (file name)
+        listen_interface: string
+        port: num
+        pidfile: string (file name)
+
+Where:
+* ``pidfile`` (string) - name of file contains pid of daemon process.
```

Comment:
It makes sense to arrange the field descriptions in the same order in which they appear in the configuration.

Reply:
Done.